app/views/people/_policy_detail.html.erb

```diff
@@ -72,7 +72,9 @@
 		<p></p>
 		<%= link_to "Cancel / Term", cancelterminate_policy_path(p), :class => "btn btn-inverse" %>
     <p></p>
-    <%= link_to "1095A Document", generate_tax_document_form_policy_path(p, {person_id: @person.id}), :class => "btn btn-primary" %>
+    <% unless p.is_shop? %>
```

Comment:
@blackeaglejs this is great. I was just wondering, is there a is_ivl? or is_individual? I think it would be better to say "is individual?"  as opposed to saying " it's not shop?". The reason is let's say there is some other market or sub market, like Coverall, that could in the future make this button show up depending on how it's defined.

Reply:
There isn't one at this time. If it's preferable, I could write that method instead and then we could use it here.

Reply:
Zoheb, if it's not too much work, I think it would be better. But I don't want to delay deployment. If you can do a quick turnaround, I think it would be best.

Reply:
I'm looking through the data, and we don't have a clean way in Glue to distinguish between coverall and IVL enrollments, which I'd want to do. Might be best to come back to this.

Reply:
This is fine for now